fuck.js

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env node
 
+/// Usage: jsfuck inputfile.js [true|false] > outputfile.js
```

Comment:
In this commit just true or false to enable/disable eval source.

The `> outputfile.js` part is just stream redirection to ouput to file what the program is writing to standard output.

It's not all that different from your old version: I just check if the argv length is 4, and convert the argument to boolean (`argv[2] === "true"`), and pass that value to the `JSFuck.encode` function.

I needed this to JSFucked files with eval source on.